### PR TITLE
Added artist resources page

### PR DESCRIPTION
**Pull request by localastronaut**
Still needs a lot of work, but thought this would be a good start

### CODE REVIEW

File: templates/extra/resources.html [hunk 1]
```diff
@@ -0,0 +1,293 @@
+{% extends "base.html" %}
+
+{% block content %}
+
```
Comment (RevolutionTech) [inline templates/extra/resources.html]: nit: Remove unnecessary newline

File: templates/extra/resources.html [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+        <div class="columns small-12">
+            <div class="columns small-2 float-left" data-sticky-container>
+                <div class="sticky" data-sticky data-anchor="example1">
+                    <Ul>
+                        <!-- Overview -->
+                        <ul class="vertical menu content-box" data-accordion-menu>
+                            <li>
+                                <a class="content-header" href="overview-nested">Overview</a>
+                                <ul class="menu vertical nested">
+                                    <li><p class="content-sub"><a href={% url 'resources' %}#overview-1>What is PerDiem</a></p></li>
```
Comment (RevolutionTech) [inline templates/extra/resources.html]: These URLs are on the same page, so you should need `{% url 'resources' %}`.

File: templates/extra/resources.html [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+            <div class="columns small-9 float-right" id="example1" data-something>
+                <div id="overview">
+                    <h3>overview</h3>
+                    <div class="break">
+                        <hr>
+                    </div>
+                    <div id="overview-1">
+                        <h4>What is PerDiem?</h4>
+                        <p>
+                        A platform where you can raise money for your music projects by having fans and people invest in your music.
```
Comment (RevolutionTech) [inline templates/extra/resources.html]: nit: Indent all of these paragraphs in, or keep them on the same line. ie.
```html
<p>
    foo
</p>
```
or
```
<p>foo</p>
```
but not
```
<p>
foo
</p>
```

File: templates/extra/resources.html [hunk 4]
```diff
@@ -0,0 +1,293 @@
...
+                        </p>
+                    </div>
+                    <div id="overview-2">
+                        <h4>Why should I let my fans invest in my music?</h4>
+                        <p>
+                        The same reason that most businesses allow people to invest in them. When you give a percentage to investors, this creates an incentive for them to fund your project. This also gives them an incentive to make your project successful. The more money you earn, the more they can earn. It's a win-win for everyone. 
+                        </p>
+                    </div>
+                    <div id="overview-3">
+                        <h4>Why Should I trust perdiem?</h4>
```
Comment (RevolutionTech) [inline templates/extra/resources.html]: Make perdiem capitalized here?

File: templates/extra/resources.html [hunk 5]
```diff
@@ -0,0 +1,293 @@
...
+                        <p>
+                        PerDiem is built on trust and integrity. We rely on our relationship with the creative community to operate. This means we must always act in the best interest of artists. 
+                        </p>
+                    </div>
+                    <div id="overview-4">
+                        <h4>How does it all work? </h4>
```
Comment (RevolutionTech) [inline templates/extra/resources.html]: nit: Remove space after question mark before `</h4>`. ie. `<h4>How does it all work?</h4>`

File: templates/extra/resources.html [hunk 6]
```diff
@@ -0,0 +1,293 @@
...
+                        <p>
+                        We are creating a new way for artists to get funding for their music while keeping creative control and making music a sustainable career choice for everyone.  We encourage artists that have been able to create their music using perdiem to help other artists as well. Together, we can create a community of creators that are all helping to push each other and create a better and more sustainable world for everyone. 
+                        </p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+{% endblock %}
```
Comment (RevolutionTech) [inline templates/extra/resources.html]: nit: Add newline at the end of file

**Comment by RevolutionTech** [comment]
To create a test for this page, add a line in this method:
https://github.com/RevolutionTech/perdiem-django/blob/master/perdiem/accounts/tests.py#L12
for the new page.

File: templates/extra/resources.html (hunk 2 above)
Comment (localastronaut) [inline templates/extra/resources.html]: "should" or "shouldn't"?

Comment (RevolutionTech) [inline templates/extra/resources.html]: Oops, yeah my bad. You shouldnt* need the url tag at all.

File: static/css/perdiem.css [hunk 7]
```diff
@@ -299,3 +299,22 @@ ul.tabs li {
     margin: 0 auto;
     padding: 20px;
 }
+
```
Comment (RevolutionTech) [inline static/css/perdiem.css]: Can these CSS styles be in their own .CSS file?
I'd rather not have common styles in `perdiem.css`

Comment (RevolutionTech) [inline static/css/perdiem.css]: rather only* have common styles

File: templates/extra/resources.html [hunk 8]
```diff
@@ -0,0 +1,286 @@
...
+                                    <li><p class="content-sub"><a href={% url 'resources' %}#next-2>Raising More</a></p></li>
+                                    <li><p class="content-sub"><a href={% url 'resources' %}#next-3>Another Campaign</a></p></li>
+                                    <li><p class="content-sub"><a href={% url 'resources' %}#next-4>Helping others</a></p></li>
+                                </ul>
+                            </li>
+                        </ul>
+                    </ul>
+                </div>
+            </div>
+            <div class="columns small-9 float-right" id="example1" data-something>
```
Comment (RevolutionTech) [inline templates/extra/resources.html]: `data-something`?

File: static/css/perdiem.css (hunk 7 above)
Comment (localastronaut) [inline static/css/perdiem.css]: Cool. How do I get it to load the new CSS file? Also, where should it be saved in the CSS folder?

Comment (RevolutionTech) [inline static/css/perdiem.css]: I would create a new folder `extras/` under the `css/` directory and in there create a new file `resources.css` with these styles. Then in the template, you can load this new CSS file by adding:
```html
{% block extrastyle %}
    <link rel="stylesheet" href="{% static "css/extra/resources.css" %}" type="text/css" charset="utf-8" />
{% endblock %}
```

This block should go before the `content` block.

Comment (localastronaut) [inline static/css/perdiem.css]: screen shot 2016-05-26 at 10 32 47 pm

Got an error when adding that code to the page

Comment (RevolutionTech) [inline static/css/perdiem.css]: You need to add `{% load staticfiles %}` at the top of the page.

File: static/css/extra/resources.css [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+    font-size: .8em;
+    padding: 0;
+    margin: 0;
+    text-align: left;
+}
+
+.content-box {
+    min-width: 125px;
+    display: inline-block;
+}
```
Comment (RevolutionTech) [inline static/css/extra/resources.css]: nit: Add newline at the end of file

File: static/css/resources.css [hunk 10]
```diff
@@ -0,0 +1,18 @@
+.content-header {
```
Comment (RevolutionTech) [inline static/css/resources.css]: I don't think you meant to commit this file (you have an identical copy in `extras/`).

File: templates/extra/resources.html [hunk 11]
```diff
@@ -0,0 +1,289 @@
+{% extends "base.html" %}
+{% load staticfiles %}
```
Comment (RevolutionTech) [inline templates/extra/resources.html]: This top section looks good, but I like to break apart the blocks by whitespace, and the extends and imports. I would prefer if it looks like this:
```html
{% extends "base.html" %}

{% load staticfiles %}

{% block extrastyle %}
    ...
{% endblock %}

{% block content %}
    ...
```

File: templates/extra/resources.html [hunk 12]
```diff
@@ -0,0 +1,289 @@
...
+        <div class="columns small-12">
+            <div class="columns small-2 float-left" data-sticky-container>
+                <div class="sticky" data-sticky data-anchor="example1">
+                    <Ul>
+                        <!-- Overview -->
+                        <ul class="vertical menu content-box" data-accordion-menu>
+                            <li>
+                                <a class="content-header" href="overview-nested">Overview</a>
+                                <ul class="menu vertical nested">
+                                    <li><p class="content-sub"><a href=#overview-1>What is PerDiem</a></p></li>
```
Comment (RevolutionTech) [inline templates/extra/resources.html]: All of your links should be surrounded by quotes.
```html
<a href="#overview-1">
```
not
```html
<a href=#overview-1>
```

As is, these links probably won't work correctly on some browsers.

File: templates/extra/resources.html [hunk 13]
```diff
@@ -0,0 +1,289 @@
...
+                        <h4>Helping other artists get funded</h4>
+                        <p>
+                            We are creating a new way for artists to get funding for their music while keeping creative control and making music a sustainable career choice for everyone.  We encourage artists that have been able to create their music using PerDiem to help other artists as well. Together, we can create a community of creators that are all helping to push each other and create a better and more sustainable world for everyone. 
+                        </p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
```
Comment (RevolutionTech) [inline templates/extra/resources.html]: nit: Remove the empty line between `</div>` and `{% endblock %}`.

File: templates/extra/resources.html [hunk 14]
```diff
@@ -0,0 +1,291 @@
...
+                            </li>
+                        </ul>
+                        <!-- Join -->
+                        <ul class="vertical menu content-box" data-accordion-menu>
+                            <li>
+                                <a class="content-header" href="overview-nested">Join</a>
+                                <ul class="menu vertical nested">
+                                    <li><p class="content-sub"><a href="#join-1">Apply To PerDiem</a></p></li>
+                                    <li><p class="content-sub"><a href="#join-2">Amount to Raise</a></p></li>
+                                    <li><p class="content-sub"><a href=#join-3">Giving Back</a></p></li>
```
Comment (RevolutionTech) [inline templates/extra/resources.html]: You missed a quote here.

**Pull request merged**